.coderabbit.yaml

```
@@ -0,0 +1,133 @@
+    scope: local
+  jira:
+    usage: disabled
+    project_keys: []
+  linear:
+    usage: disabled
+    team_keys: []
+  pull_requests:
+    scope: global
+code_generation:
+  docstrings:
+    language: en-US
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)